ros-hsrb_moveit_config/install.yaml

```diff
@@ -1,5 +1,4 @@
 - type: ros
   source:
     type: git
-    url: git@github.com:tue-robotics/hsrb_moveit.git
-    sub-dir: hsrb_moveit_config
+    url: https://github.com/tue-robotics/hsrb_moveit_config.git
```

Comment:
should we rename the repo hero_moveit_config?

Reply:
This is a valid question. I'd keep the repo as close as possible to the original to allow keeping it upstream. Nevertheless, you could consider renaming the target to allow an easy switch back if desired. I'm not sure of the use case yet.

Reply:
I would keep the package name and the repo the same. And as the launch files link to config files and other launch files in `hsrb_moveit_config`, you would create a significant diff to upstream. Which will make it difficult to interact with upstream.
So I suggest to keep it this way.